src/app/search/search.component.ts

```diff
@@ -1,6 +1,8 @@
-import {Component, OnInit, ViewChild, enableProdMode} from '@angular/core';
-import {Client} from 'elasticsearch';
-import {CommunicatorService} from '../shared/communicator.service';
+import * as bodybuilder from 'bodybuilder';
```

Comment:
Not sure if Javascript works the same way as Java, but a star import could complicate refactoring and dependency clean-up